Extension/package.json

```diff
@@ -1459,7 +1459,7 @@
   },
   "scripts": {
     "compile": "npm run vscode:prepublish",
-    "compileDev": "webpack --mode development",
+    "compileDev": "node ./tools/prepublish.js && webpack --mode development",
```

Comment:
Can we refactor out the `node ./tools/prepublish.js` so if that commandchanges we don't have to remember to change it in two places?